### DISCUSSION TITLE
Select Component showing unmatching results on Searching

### DISCUSSION THREAD

**Discussion by BishwayanGhosh** [formio/formio.js#5851] [Q&A]
Hello,
I am currently using Select component of Form.io.

When I search for any keyword in the select component, I get a suggestion list which has - 
   1. Suggestions which "CONTAIN" the searched keyword. (ex: 'CARE' being suggested when you look for CAR)
   2. Suggestions, which do not exactly CONTAIN the keyword, but kind of has some similarity index(ex: 'CHAIR' being also suggested while you look for CAR - most probably because it has the characters of 'C','A','R'). - which is not required in my scenario. Is there any way to disable this logic?

Please let me know, if there is any way to filter only by the CONTAIN logic(as mentioned in pt1).

**Comment by lane-formio** [comment]
There's a configuration option under the data tab, "use exact search".
![image](https://github.com/user-attachments/assets/144f72b5-aeb2-4b0c-b824-3715573e05af)

**Reply by BishwayanGhosh** [reply]
Thanks for the suggestion. I tried out the same. But it still behaves the same. It is still returning SIMILAR suggestions, instead of returning the EXACT matches.
Attached is a snap of my config-|
![image](https://github.com/user-attachments/assets/a278481f-5e92-4427-b7a2-aec0cb145902)

**Reply by lane-formio** [reply]
I'll try to get this checked out.

**Reply by ojo24** [reply]
Hello @lane-formio I am facing the same issue, did you happen to get any solution for this?
